### DISCUSSION TITLE
[Sophia Script evolution] deactivation of services

### DISCUSSION THREAD

**Discussion by nreljed** [farag2/Sophia-Script-for-Windows#268] [Q&A] [answered]
I just discovered this script by accident when typing "debloat win 10 or 11 " in google
I test it and it works perfectly on freshly installed win 10 and 11.
the performance gain is noticeable but ...
I notice that I still have so many useless services (70)+ in windows task manager 
for example "windows search" which I systematically deactivate..

I dont know once i used a script from another github and i end up with 30+ services which is fantastic for old pc or pure pc gamer profiles..

the normal method before (scripts and sophia) was [Easy Service Optimizer v1.2 ](https://www.sordum.org/8637/easy-service-optimizer-v1-2/)
but unfortunately as you can see the program is totally outdated  and dead..  not ready to be updated despite the constant demand from people to update it... and even if it is updated ..sophia just killed it now :) rip ESO

my idea  is to integrate a solid database of useless services to deactivate in sophia scripts. and definitely go to the next level to be  "world best debloater ever"

**Comment by farag2** [comment, marked as answer]
> I notice that I still have so many useless services (70)+

You don't. All services are necessary.

> which is fantastic for old pc or pure pc gamer profiles..

This doesn't have an effect on OS responsiveness

> my idea is to integrate a solid database of useless service

There is only `1` service that can be disabled without any harmness: diagtrack. None of otheres can be touched, unfortunately. Don;t believe anyone that you can disable in the OS this oe that without any implications: all service have their appropriate dependencies. So don;t play with them. :) So I provide only one service to disable being confident that everything will be working stable after that. So it's only on your conscience disabling everything you googled in YouTube.
